README.md

```diff
@@ -129,7 +142,7 @@ Let's say the Posts in the previous question are available at `http://localhost:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
+
 ```
```

Comment:
When you have time, I would recommend reviewing the following lesson plan: https://github.com/ga-wdi-lessons/js-ajax
```
$("button").on("click", () => {
  var url = "http://localhost:3000/posts"
  $.ajax({
    url: url,
    type: "get",
    dataType: "json"
  }).done((response) => {
    console.log(response)
  }).fail(() => {
    console.log("Ajax request failed!")
  }).always(() => {
    console.log("This always happens regardless of successful ajax request or not.")
  })
})
```